dashboards-observability/public/components/event_analytics/explorer/sidebar/sidebar.scss

```diff
@@ -135,4 +135,8 @@
       }
     }
   }
+}
+
+.ws__configPanel--right{
+  overflow-y: unset;
```

Comment:
Add extra line.

Reply:
I think that is already added. Github shows warning for that, as you can see on the earlier file.